.vscode/settings.json

```diff
@@ -21,5 +21,9 @@
   "prettier.semi": false,
   "prettier.singleQuote": true,
   "prettier.trailingComma": "es5",
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+
+  "[markdown]": {
```

Comment:
This is One Weird Trick in case other extensions in the repository that get caught by eager auto-formatting.

Reply:
$USER